eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/listeners/GenericProgressListener.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2014-2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.accession.dbsnp.listeners;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.listener.StepListenerSupport;
+import org.springframework.batch.core.scope.context.ChunkContext;
+
+import java.util.List;
+
+public class GenericProgressListener<I, O> extends StepListenerSupport<I, O> {
```

Comment:
This class could be moved to `variation-commons-batch`, as I suspect we will use it quite often. This can be done after the PR is merged though, or even as a separate refactoring ticket, because it is low priority.

Reply:
agree

Reply:
Added TODO comment to track this.